### ISSUE TITLE
is there any problem with function normalize in img_utils.py 

### ISSUE THREAD

**Issue by yzg050215** [ycszen/TorchSeg#8]
def normalize(img, mean, std):
    # pytorch pretrained model need the input range: 0-1
    img = img - mean
    img = img.astype(np.float32) / 255.0
    # img = img / std

    return img

hi ycszen, i feel that there is a problem about the normalize function, the img passed into the function is lied in 0-255, while the mean found in the config.py is typically 0.4-0.5, so if the normalize should be changed into  
 
def normalize(img, mean, std):
    # pytorch pretrained model need the input range: 0-1
   
    img = img.astype(np.float32) / 255.0
    img = img - mean
    # img = img / std

    return img

thank you very much

**Comment by ycszen** [comment]
Yes. You are right. It's my fault with the careless uploading. I have corrected it.
